### PR TITLE
add language on public ids; example of urn

### LINKED ISSUE #1432: first example contains an http url identifying a credential
https://w3c.github.io/vc-data-model/#example-a-simple-example-of-the-contents-of-a-verifiable-credential

```json
{
  
  "@context": [
    "https://www.w3.org/ns/credentials/v2",
    "https://www.w3.org/ns/credentials/examples/v2"
  ],
  
  "id": "http://university.example/credentials/1872", // bad idea to recommend this
  
  "type": ["VerifiableCredential", "ExampleAlumniCredential"],
  
  "issuer": "https://university.example/issuers/565049",
  
  "validFrom": "2010-01-01T19:23:24Z",
  
  "credentialSubject": {
    
    "id": "did:example:ebfeb1f712ebc6f1c276e12ec21",
    
    "alumniOf": {
      
      "id": "did:example:c276e12ec21ebfeb1f712ebc6f1",
      
      "name": "Example University"
    }
  }
}
```

### PR DESCRIPTION
**Pull request by decentralgabe**
Fix #1432

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/pull/1466.html" title="Last updated on Apr 2, 2024, 1:55 PM UTC (564e3c2)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/1466/25c3efa...564e3c2.html" title="Last updated on Apr 2, 2024, 1:55 PM UTC (564e3c2)">Diff</a>

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6704,10 +6716,6 @@ <h4>Holder</h4>
 Validation is the process by which verifiers apply business rules to
 evaluate the propriety of a particular use of a [=verifiable credential=].
         </p>
-        <ul>
```
Comment (decentralgabe) [inline index.html:6707]: this was resulting in an extra bullet point in the middle of nowhere!

File: index.html [hunk 2]
```diff
@@ -5845,14 +5847,21 @@ <h3>Data Theft</h3>
...
-individuals, such as [=holders=] and [=subjects=]. Storing this data
-can create honeypots of sensitive data that attackers are motivated to
-break into in order to acquire and exchange that data for financial gain.
-        </p>
+individuals, such as [=holders=] and [=subjects=]. The storage and
+acessibility of this data can inadvertently create honeypots of 
+sensitive data for malicious actors. These adversaries often seek to
+exploit such resevoirs of sensitive information, aiming to
+acquire and exchange that data for financial gain.
+        </p>to
```
Comment (iherman) [inline index.html:5855]:
```suggestion
        </p>
```

**Comment by brentzundel** [comment]
Thank you for this PR. I like the changes but would recommend tweaking the language to avoid any normative words (e.g., must, should) even though you aren't using them normatively. This would bring your text in line with the rest of the document which similarly avoids such words when they aren't used normatively.

File: index.html [hunk 3]
```diff
@@ -1375,13 +1375,15 @@ <h3>Identifiers</h3>
...
-Where privacy is a strong consideration, the `id` [=property=]
-MAY be omitted. Some use cases do not require, or explicitly require omitting,
-the `id` [=property=].
+scenarios. There are also other types of access and correlation mechanisms documented
+in Section <a href="#privacy-considerations"></a> that create privacy concerns.
+Where privacy is a strong consideration, it is permissible to omit the 
+`id` [=property=]. Some use cases do not require, or explicitly require omitting,
+the `id` [=property=]. Similarly, special attention is to be given to the choice between
+publicly resolvable URLs and other forms of identifiers. Publicly resolvable URLs can
+facilitate ease of verification and interoperability, yet they may also inadvertently
```
Comment (dlongley) [inline index.html:1384]:
```suggestion
facilitate ease of verification and interoperability, yet they might also inadvertently
```

File: index.html [hunk 4]
```diff
@@ -1375,13 +1375,15 @@ <h3>Identifiers</h3>
...
 <a href="#identifier-based-correlation"></a> carefully when considering such
-scenarios. There are also other types of correlation mechanisms documented in
-Section <a href="#privacy-considerations"></a> that create privacy concerns.
-Where privacy is a strong consideration, the `id` [=property=]
-MAY be omitted. Some use cases do not require, or explicitly require omitting,
-the `id` [=property=].
+scenarios. There are also other types of access and correlation mechanisms documented
+in Section <a href="#privacy-considerations"></a> that create privacy concerns.
+Where privacy is a strong consideration, it is permissible to omit the 
+`id` [=property=]. Some use cases do not require, or explicitly require omitting,
```
Comment (dlongley) [inline index.html:1381]:
```suggestion
`id` [=property=]. Some use cases do not need, or explicitly need to omit,
```

**Review by dlongley — APPROVED** [review]
Some suggestions to remove normative language, otherwise, +1.

File: index.html [hunk 5]
```diff
@@ -5845,14 +5847,21 @@ <h3>Data Theft</h3>
...
         <p>
 [=Issuers=] are advised to retain the minimum amount of data
 necessary to issue [=verifiable credentials=] to [=holders=] and
-manage the status and revocation of those credentials.
+manage the status and revocation of those credentials. Similarly,
+[=issuers=] are advised to avoid the practice of creating publicly
+resolvable credentials that include personally identifiable information
+(PII) or other sensitive data. Instead, a preference should be shown
+for credentials that are safeguarded by robust access control measures,
+ensuring that they remain inaccessible to unauthorized entities.
```
Comment (msporny) [inline index.html:5864]:
```suggestion
(PII) or other sensitive data. Software implementers are advised 
to safeguarded [=verifiable credentials=] using robust consent 
and access control measures, ensuring that they remain 
inaccessible to unauthorized entities.
```

**Review by msporny — APPROVED** [review]

**Comment by msporny** [comment]
Editorial, multiple reviews, changes requested and made, no objections, merging.

**Pull request merged**